src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -659,6 +659,15 @@ public void init() {
         Platform.runLater(() -> stateManager.focusOwnerProperty().bind(
                 EasyBind.map(mainStage.getScene().focusOwnerProperty(), Optional::ofNullable)));
 
+        EasyBind.subscribe(tabbedPane.getSelectionModel().selectedItemProperty(), selectedTab -> {
```

Comment:
Here, I used `EasyBind#subscribe` instead of binding because a bound value cannot be set, but I'm setting it in another place. See the other comment.